src/plugins/intel_gpu/tests/unit/test_cases/gemm_gpu_test.cpp

```diff
@@ -1499,7 +1499,7 @@ INSTANTIATE_TEST_SUITE_P(gemm_gpu, gemm_onednn_ndims, ::testing::ValuesIn(std::v
     gemm_onednn_test_params{ CASE_GEMM_ONEDNN_I8_6D },
 }));
 
-TEST(gemm_onednn, impl_replacement_with_cldnn) {
+TEST(gemm_onednn, DISABLED_impl_replacement_with_cldnn) {
```

Comment:
Why disabled? I think such test still make sense, gemm may be replaced with FC primitive if needed.

Reply:
I reverted this change. Thank you.